modules/visual_script/editor/visual_script_property_selector.h

```diff
@@ -60,32 +102,118 @@ class VisualScriptPropertySelector : public ConfirmationDialog {
 	String selected;
 	Variant::Type type;
 	String base_type;
+	String base_script;
 	ObjectID script;
 	Object *instance;
 	bool virtuals_only;
-	bool seq_connect;
-	VBoxContainer *vbc;
-
-	Vector<Variant::Type> type_filter;
+	VBoxContainer *vbox;
```

Comment:
Fix later. set to nullptr

Reply:
Sorry if this was confusing. I mean that the variables should be intitialized to like nullptr or 0, but since there's like 15 variables to check it can be left to another commit.